packages/wdio/src/component/type/selector-methods.type.ts

```diff
@@ -0,0 +1,3 @@
+// @ts-expect-error We ignore prefix, how we can avoid TS error?
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export type SelectorMethods<T extends string> = T extends `${infer Prefix}.${infer Selector}` ? `${Selector}` : never;
```

Comment:
Have you tried to do `infer _`?

Also, maybe to make it more universal, if selector does not follow `Prefix.Selector` pattern we should just support `Selector` pattern? 

The code I propose is this:

```
export type SelectorMethods<T extends string> = T extends `${infer _}.${infer Selector}` ? `${Selector}` : T;
```

Reply:
Actually, I think TS now supports something like this:

```
T extends `${string}.${infer Selector}`
```

Reply:
I do want to give ability for wrong selector Enum value naming, so we will stick with `never`